Dockerfile

```diff
@@ -1,4 +1,23 @@
 FROM bitnami/spark:3.5.1
 
+RUN export ORI_USER=$(id -u)
+# Switch to root to install packages
+USER root
 
-ENTRYPOINT ["sleep 10s"]
+ENV PYTHON_VER=python3.11
```

Comment:
Now I think about it I'm pretty sure bitnami installs python in the image. Is there a reason you're not using that python, assuming I'm right? If the python version here doesn't match what's in the cluster the job won't run

Reply:
yea. I can use python from the image. But I remember you want to control which python version we are using. I have no strong option on that. I am fine either way.

Reply:
In the prior repo the version was set for both the cluster and the test image (assuming you were using the same image). Here the python version could be different for the cluster and the notebook which would break things

Reply:
👍 Switched to use image python. Both base image has python installed. I don't see the differences.

Reply:
> Both base image has python installed.

What do you mean by both base images?

>  I don't see the differences.

Not sure what you mean here either

Reply:
both `bitnami/spark` and the image you used previously has python. I don't see why we spent a lot of time trying to install python before. We could just use the python form the image before.

Reply:
The old image had 3.10. If it had 3.11 I wouldn't have bothered

Reply:
👍